app/harvesters/etree_harvester_raw_result.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+from xml.etree import ElementTree
+from app.harvesters.abstract_harvester_raw_result import AbstractHarvesterRawResult
+
+
+@dataclass(kw_only=True)
+class EtreeHarvesterRawResult(AbstractHarvesterRawResult[str, ElementTree]):
```

Comment:
Etree is the specific name of a python XML library : the class should be names XMLHarvesterRawResult for naming consistency (JsonHarvesterRawResult, RdfHarvesterRawResult...)